packages/catalog-model/src/schema/kinds/Template.v1beta2.schema.json

```diff
@@ -141,14 +146,20 @@
                   "description": "A list of external hyperlinks, typically pointing to resources created or updated by the template",
                   "items": {
                     "type": "object",
-                    "required": ["url"],
+                    "required": [],
                     "properties": {
                       "url": {
                         "type": "string",
                         "description": "A url in a standard uri format.",
                         "examples": ["https://github.com/my-org/my-new-repo"],
                         "minLength": 1
                       },
+                      "entityRef": {
+                        "type": "string",
+                        "description": "An entity reference to an entity in the catalog.",
+                        "examples": ["Component:default/my-app"],
+                        "minLength": 1
+                      },
```

Comment:
We could narrow down the schema to have either an `url` or an `entityRef` and marking both as required. That would be fine as we either expect a value or a handlebars template.

We should consider to document the deprecated top level `entityRef` and `remoteUrl` too. Or remove them completely. 

The example above uses `catalogInfoUrl` as an output and should be adjusted to the current output schema.